modules/ROOT/pages/provisioning-exoscale.adoc

```diff
@@ -0,0 +1,64 @@
+= Provisioning Fedora CoreOS on Exoscale
+
+This guide shows how to provision new Fedora CoreOS (FCOS) instances on https://exoscale.com[Exoscale] Cloud Hosting.
+
+== Prerequisites
+
+Before provisioning a FCOS instance, you can use or not an Ignition configuration file containing your customizations. If you need one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to an Exoscale account. https://portal.exoscale.com/register[Register] if you don't have one.
+
+== About images
+
+Fedora CoreOS is updated at every major change on our platform. The future goal is to have Exoscale FCOS in https://getfedora.org/en/coreos/download?tab=cloud_launchable[Cloud Lauchable] section, to be updated at every release.
+
+== Launching a VM instance
+
+You can provision a FCOS instance using the Exoscale https://portal.exoscale.com/compute/instances/add?os-group=coreos[Web Portal]. Then you can choose your FCOS version and follow the instance creation wizard.
```

Comment:
Can we move this sentence to the bottom of the page as its own section? We do prefer highlighting flow which are automation-friendly first.

Reply:
@lucab just to be transparent, there is two way to deploy FCOS at Exoscale, we provide an official image template, which is `Linux Fedora CoreOS 32 64-bit`, Launchable from the platform: https://www.exoscale.com/templates#Fedora

or by uploading the qcow2 as a custom image and launch it like an official image template